src/xenia/ui/imgui_drawer.cc

```diff
@@ -142,10 +142,12 @@ void ImGuiDrawer::Initialize() {
   InitializeFonts();
 
   auto& style = ImGui::GetStyle();
-  style.ScrollbarRounding = 0;
-  style.WindowRounding = 0;
-  style.TabRounding = 0;
-  style.Colors[ImGuiCol_Text] = ImVec4(0.89f, 0.90f, 0.90f, 1.00f);
+  style.ScrollbarRounding = 8.0f;
```

Comment:
Previously it was set to 6.f now you set it to 8.f why? It was rounded enough in my opinion. PS. Check how it looks with achievement notification